### PR TITLE
docs/: initial deployment config and files

### PR DESCRIPTION
**Pull request by artur-rs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,98 @@
...
+### Local build
+
+```shell
+virtualenv -p $(which python3) venv
+source venv/bin/activate
+pip install -r requirements.txt
+mkdocs serve
+```
+
+By default, it will host a local copy od documentation at:
```
Comment (pietrushnic) [inline README.md:19]: Spellchecker

**Review by pietrushnic — COMMENTED** [review]
Some of that will quickly become obsolete maybe we should add 3mdeb contribution guidelines? Those are essentially the same across all markdowns and mkdocs projects which we have a few.

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+name: ci
```
Comment (pietrushnic) [inline .github/workflows/deploy.yml:2]: I'm pretty sure @macpijan had some generic pre-commit integration project.

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+Zarhus isn't just an operating system; it's your digital chess knight, making
+moves that matter. Experience the innovation that comes with a system designed
+to adapt, secure, and conquer the complexities of the ever-evolving embedded
+solutions.
+
+### 🌐 Your Digital Landscape, Secured:
+
+Just as the knight secures the chessboard, Zarhus safeguards your embedded
+systems. Navigate through the intricate ecosystem of hardware and software with
+ease, confident in the adaptability and resilience that Zarhus provides.
```
Comment (pietrushnic) [inline docs/index.md:32]: Over time we would look for more practical, factual and number driven reasons. Let's say it is OK for now now despite little BLIPpy

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+## 🌍 Community
+
+Become a part of the Zarhus community:
+
+- **Chat with Us**: Join the conversation in the [Dasharo Matrix
+  Workspace](https://matrix.to/#/#dasharo:matrix.org) where you can ask about
```
Comment (pietrushnic) [inline docs/index.md:39]: We can easily create matrix space for Zarhus on matrix.3mdeb.com

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  the Zarhus OS, Dasharo Tools Suite and more!
```
Comment (pietrushnic) [inline docs/index.md:40]: meta-rte?

**Review by pietrushnic — APPROVED** [review]
@artur-rs I love it, very good writing. Explanation of business.model and how to get commercial support is missing but it can be next step.

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (artur-rs) [reply .github/workflows/deploy.yml:2]: @pietrushnic pre-commit and SEO yml will be added later, I didn't want to add everything on initial deploy

File: README.md (hunk 1 above)
Comment (artur-rs) [reply README.md]: fix: 2802bad95ec3b4d17e476d5581d7bb50e47fe417

File: docs/index.md (hunk 5 above)
Comment (artur-rs) [reply docs/index.md]: fix: 2802bad95ec3b4d17e476d5581d7bb50e47fe417

**Comment by artur-rs** [comment]
@pietrushnic as the PR is approved, I'll merge it for the deployment test for `gh-pages`. All remaining comments are aggregated in the next step Jira task.

**Pull request merged**